### DISCUSSION TITLE
Date Range Picker

### DISCUSSION THREAD

**Discussion by ghost** [solidjs/solid#874] [Q&A] [answered]
Hi,

Has anyone build a date range picker (or adapted one like https://reactdatepicker.com/#example-date-range) for SolidJS? I can't find one and it's the last thing I need for my SolidJS PWA app and I would so love not haveing to build it from scratch!

Jim

**Comment by bannndi** [comment, marked as answer]
I used Flatpickr in my SolidJS app.

**Reply by ghost** [reply]
Awesome - works great! Thanks :)

**Reply by ghost** [reply]
onMount(() => {
    flatpickr("#dateRange", {
      altInput: true,
      altFormat: "F j, Y",
      dateFormat: "Y-m-d"
    });
  });

**Comment by rnwonder** [comment]
Created one https://github.com/rnwonder/solid-date-picker
